### PR TITLE
SRVKS-552: Add information about serverless configuration options

### PR DESCRIPTION
**Pull request by abrennan89**
Jira: https://issues.redhat.com/browse/SRVKS-552

Direct preview link(s):
- https://deploy-preview-40280--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/serverless-configuration

Applies for OCP 4.6+
_(Manual cherrypick probably required for 4.6 due to topic map differences)_

### CODE REVIEW

**Review by cardil — APPROVED** [review]
/lgtm

from QE

File: serverless/admin_guide/serverless-configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-configuration"]
+= Configuring {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: serverless-configuration
+
+toc::[]
+
+The {ServerlessOperatorName} manages the *global* configuration of a Knative installation, including propagating values from the `KnativeServing` and `KnativeEventing` custom resources to system link:https://kubernetes.io/docs/concepts/configuration/configmap/[config maps].
```
Comment (stevsmit) [inline serverless/admin_guide/serverless-configuration.adoc:9]: Usually asterisks are used for GUI items like menus, buttons, page titles, etc., as per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting. I'm not entirely sure, but is this one of those?

Comment (abrennan89) [reply serverless/admin_guide/serverless-configuration.adoc:9]: I think I actually did this by mistake :) I'll fix it to be italics, which would be correct for a first instance of a term, is that right?

**Pull request merged**